pytest/tests/sanity/rosetta.py

```diff
@@ -350,6 +390,65 @@ def setUpClass(cls) -> None:
     def tearDownClass(cls) -> None:
         cls.node.cleanup()
 
+    def test_zero_balance_account(self) -> None:
+        test_amount = 10**22
```

Comment:
Since this is testing zero balance accounts, wouldn't it make more sense to create an account with (close to) zero balance? If I understand it correctly, that would also trigger the arithmetic overflow that you saw in production.

But I guess you also want to test the liquid balance amount for the case where we add more keys and suddenly it's no longer a zero-balance account. Maybe make it two separate tests? Or create it with low balance, check the liquid amount is correct them add more funds and add more keys.

Reply:
The new version of the test first transfers 1 yoctoNEAR (to create the account) and checks that the balance is accessible.